app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+    sours_fl, new_fl = command.split()[0], command.split()[2]
```

Comment:
Старий файл буде після спліта другим (індекс 1)
Розпакуй результат Спліта вірдразу у змінні ось так якось
```suggestion
    cmd, sours_fl, new_fl = command.split()
```

Reply:
Sorry, не зовсім зрозумів про другу дію, що означає "розпакуй результат Спліта відразу у змінній" і що означає cmd?

Reply:
В тебе команда буде мати формат "cp file1t.txt file2.txt", якщо ти зробиш спліт будеш мати ("cp", "file1.txt", "file2,txt")
Вот це тюпл пайтон вміє розпакувати відразу у зміні, як я показав